### ISSUE TITLE
Account Skipping, when changing Accounts

### ISSUE THREAD

**Issue by ghost** [MetaMask/metamask-extension#8905]
UX Bug:

I can no longer select my account 5 on metamask when scrolling down it skips that account so I can only see the accounts below it.

I am unable to access this account as I cannot select it.

I found a work around:

If you highlight the text in the accounts, the user can then scroll down once highlighted.

**Comment by Gudahtt** [comment]
Thanks for the report @NovedNav! I had a few questions to help with investigating this:

* Which version of MetaMask were you using?
* How many accounts did you have when you encountered this?
* Which browser were you using?

**Comment by NovedNav** [comment]
version 7.7.9
9 accounts
chrome

**Comment by Gudahtt** [comment]
@NovedNav How are you scrolling? Are you using the mouse wheel, or relying upon the 'Down' button shown, or some other means?

I am able to scroll down using the mouse wheel without issue. I'm not sure how I'd scroll without a mouse wheel though, and I haven't tested this on a device with a touch screen.

**Comment by NovedNav** [comment]
I am using the arrow down button on MetaMask. I do not have a scroll mouse to use.  

The issue is once I push the down button it goes down unevenly so it skips numbers so the down button does not allow me to select certain addresses.

**Comment by Gudahtt** [comment]
Thanks @NovedNav , I understand now. We should provide a way to navigate the account list with either a keyboard or a mouse lacking a scroll wheel. I'll speak to our design team about how we can improve this.

**Comment by Gudahtt** [comment]
I did speak to the design team shortly after my last comment, but forgot to reply here. We decided that we should fix this by showing the scrollbar.

**Comment by BboyAkers** [comment]
I can look into this! 🙂

**Comment by BboyAkers** [comment]
PR is up for this! 
https://github.com/MetaMask/metamask-extension/pull/11963
